flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Nix flake for deployversioner";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
```

Comment:
Why this old package version. ?
Latest right now is 24.11

Reply:
Just copy-paste. I'll update to latest